File: Java/src/URI/URI1020.java
package URI;

import java.util.Scanner;

public class URI1020 {

	public static void main(String[] args) {

		Scanner input = new Scanner(System.in);

		int idadeDias = input.nextInt();

		int anos = idadeDias / 365;
		idadeDias %= 365;

		int meses = idadeDias / 30;

		int dias = idadeDias % 30;

		System.out.printf(anos + " ano(s)%n" + meses + " mes(es)%n" + dias + " dia(s)%n");

		input.close();
	}

}

File: Python/URI1020.py
idadeDias = int(input())

anos = idadeDias//365
idadeDias = idadeDias - (anos*365)

meses = idadeDias//30
idadeDias = idadeDias - (meses*30)

dias = idadeDias

print("%s ano(s)" %(anos))
print("%s mes(es)" %(meses))
print("%s dia(s)" %(dias))
